go.mod

```diff
@@ -4,14 +4,14 @@ go 1.21
 
 require (
 	github.com/Azure/go-ntlmssp v0.0.0-20221128193559-754e69321358
-	github.com/alecthomas/chroma v0.10.0
+	github.com/alecthomas/chroma v0.11.1
 	github.com/danwakefield/fnmatch v0.0.0-20160403171240-cbb64ac3d964
 	github.com/dlclark/regexp2 v1.10.0
 	github.com/gandarez/go-olson-timezone v0.1.0
 	github.com/gandarez/go-realpath v1.0.0
 	github.com/juju/mutex v0.0.0-20180619145857-d21b13acf4bf
 	github.com/kevinburke/ssh_config v1.2.1-0.20220605204831-a56e914e7283
-	github.com/matishsiao/goInfo v0.0.0-20210923090445-da2e3fa8d45f
+	github.com/matishsiao/goInfo v0.1.0
```

Comment:
There's no `v0.1.0` for https://github.com/matishsiao/goInfo

Reply:
We're overwriting it with our fork, so i'm just matching that version.